app/src/components/governance/ProposalHeader.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <header class="header">
+    <div class="header-top">
+      <div>
+        <button><i class="material-icons">arrow_back</i>Proposals</button>
+      </div>
+      <div>
+        <span v-if="votingPeriod">Voting Period</span>
+        <span v-if="depositPeriod">Deposit Period</span>
+      </div>
+      <div>
+        <button><i class="material-icons">link</i>Link</button>
+        <button><i class="material-icons">share</i>Twitter</button>
+        <button>Vote</button>
+      </div>
+    </div>
+    <h2>{{ title }}</h2>
+    <div>
+      <div>
+        Proposer:
```

Comment:
proposer is a validator? should we use an existing component/structure to display that?

Reply:
i don't think we have a component like this but i can make one

Reply:
We display the image and the name together in the transaction items